components/reducers/entityReducer.js

```diff
@@ -104,5 +106,10 @@ export const entityReducer = (state, action) => {
         ...state,
         modalContent: action.payload,
       };
+    case SET_VALIDATE_ALL:
+      return {
+        ...state,
+        all_validates: action.payload
```

Comment:
The naming conventions are inconsistent (some are camelCase while others are snake_case).

For example: 
```
export const initialState = {
  show: false,
  entity_type_id: null, (snake_case)
  modalContent: null, (camelCase)
  all_validates: null, (snake_case)
};
```

For javascript constants, variables, functions, etc, it's better to use camelCase.

For any database related variables / constants, it's better to use snake_case (if you will be using APIs, query routes, etc)

In this case, it's better to use camelCase.

e.g: 
```
export const initialState = {
  show: false,
  entityTypeId: null, (snake_case)
  modalContent: null, (camelCase)
  allValidates: null, (snake_case)
};
```